docs/06-cpu-throttling.md

```diff
@@ -14,10 +14,6 @@ For details of the feature, see this [blog
 post](https://www.cloudfoundry.org/blog/better-way-split-cake-cpu-entitlements/)
 and this [video](https://youtu.be/vV87xmxKLeA).
 
-## Enabling and disabling cpu-throttling
```

Comment:
this header still needs to be here, just the sentence below about the flag needs to be deleted.